### PR TITLE
Add gem for CircleCI deployment to S3

### PR DESCRIPTION
**Pull request by JohnWeaver**
To allow CircleCI to automatically deploy the Learning Hub to an S3 bucket on AWS on Master builds.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,9 @@ source 'http://rubygems.org'
...
 # gem 'redcarpet'
-
-# Added at 2018-01-15 15:18:35 +0000 by root:
-gem "jekyll-pagination", "~> 0.0.4"
+
+# Added at 2018-01-15 15:18:35 +0000 by root:
+gem "jekyll-pagination", "~> 0.0.4"
+
+# Added at 2018-01-30 by JohnWeaver // Allows CircleCI to upload to S3
+gem 's3_website '
```
Comment (skinofstars) [inline Gemfile]: i'm guessing this shouldn't have a space after the  name. would also be nice to lock it to a version number.

**Comment by skinofstars** [comment]
this commit should have an updated lock file with it

File: Gemfile (hunk 1 above)
Comment (JohnWeaver) [reply Gemfile]: @skinofstars You are correct on the space.
Current version is v3.4.0, I'm not sure where CircleCI gets gems from, am I safe to use that version?

Comment (skinofstars) [reply Gemfile]: if you add the gem using the command `bundle add s3_website`, it'll both add the latest version and update the lock file

Comment (JohnWeaver) [reply Gemfile]: I'm working from the web interface, not sure where you want the command to be run? Please advise?

**Pull request merged**